hs_core/templates/pages/genericresource.html

```diff
@@ -1587,9 +1587,11 @@ <h4 class="modal-title">Confirm file deletion</h4>
                             <div class="tab-pane" id="relevantTools">
                             {% if not metadata_form %}
                                 {% for tool in relevant_tools %}
-                                    <h4>{{ tool.title }}</h4>
-                                    <a href="{{ tool.url }}" target="_blank"><button class="btn btn-primary">Launch App</button></a>
-                                    <hr>
+                                    <div style="display:inline-block; padding-left:20px; text-align:center;">
+                                        <h4>{{ tool.title }}</h4>
+                                        <a href="{{ tool.url }}" target="_blank"><img src="{{ tool.icon }}" style="width:100px; height:100px;"
```

Comment:
@hyi How do I merge changes made to the hydroshare/static/css/hydroshare_core.css file? It appears that this file is not actually stored in the GitHub repository but is actually pulled from somewhere during `./hsctl rebuild`. Or are you suggesting that I create a new .css file in hs_core/static/css?